Formula/ats2-postiats.rb

```diff
@@ -12,11 +12,33 @@ class Ats2Postiats < Formula
   end
 
   depends_on "gmp"
+  depends_on "json-c" => :recommended
+  depends_on "bdw-gc" => :optional
 
   def install
     ENV.deparallelize
+    ENV["PATSHOME"] = "#{lib}/ats2-postiats-#{version}"
+    ENV["PATH"] = ENV["PATSHOME"] + "/bin:" + ENV["PATH"]
+
     system "./configure", "--prefix=#{prefix}"
-    system "make", "all", "install"
+
+    args = `%W`
```

Comment:
That line is wrong, and leads to build failure `Error: No such file or directory - %W`